cmd/project/docker_up.go

```diff
@@ -37,7 +37,7 @@ var dockerUpCmd = &cobra.Command{
 			return err
 		}
 
-		return runTransparentCommand(exec.CommandContext(cmd.Context(), "docker", "compose", "up", "-d", "--wait", "--remove-orphans"))
```

Comment:
this looks fishy

Reply:
Shall we use `up` by default and enable `watch` via a flag instead?

Reply:
does watch also start the containers?

Reply:
Yes, it does